debian/jitsi-meet-prosody.postinst

```diff
@@ -154,6 +154,13 @@ case "$1" in
             PROSODY_CONFIG_PRESENT="false"
         fi
 
+        # Old versions of jitsi-meet-prosody come with the extra plugin path commented out (https://github.com/jitsi/jitsi-meet/commit/e11d4d3101e5228bf956a69a9e8da73d0aee7949)
+        # Make sure it is uncommented, as it contains required modules.
+        if grep -q '--plugin_paths = { "/usr/share/jitsi-meet/prosody-plugins/" }' ;then
```

Comment:
Missing `--` and missing filename:
```
grep: unrecognized option '--plugin_paths = {
+"/usr/share/jitsi-meet/prosody-plugins/" }'
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
```

Reply:
I'm sorry about that, obviously my change wasn't tested as much as it should have been. Fixed now.

Reply:
No problems (that's what beta testers are for) and thanks for providing and caring for these debian packages!

Reply:
Thanks for the fix. It's been 15 days and upgrading jitsi-unstable still breaks running systems. This is although unstable has been updated since then (if I understand my update logs correctly).

Could someone please make sure this small fix of a regression is accepted for unstable?  Many thanks.

Reply:
@ned64 how does it break it, this was already fixed in master.

Reply:
https://github.com/jitsi/jitsi-meet/commit/f6df76ab10a9edc59c14b44cfe85dd50636143bb

Reply:
@damencho I just experimented with my config with the help of the original bug report.  It turns out that my `/etc/prosody/conf.avail/meet.example.org/` had an invalid plugin path: `plugin_paths = { "/jitsi/sites/meet.example.org/prosody-plugins/" }`.  That made the `postinst` script bail out.  It never happened before (non-existing path was silently ignored).

Reply:
Perhaps this is a related but different issue?  Shall I file it?